exercises/triangle/.meta/template.j2

```diff
@@ -0,0 +1,14 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header(["equilateral", "isosceles", "scalene"]) }}
+
+{% for case in cases -%}
+class {{ case["description"] | camel_case }}Test(unittest.TestCase):
+    {% for case in case["cases"] %}
+    def test_{{ case["description"] | to_snake }}(self):
+        input = {{ case["input"]["sides"] }}
+        expected = {{ case["expected"] }}
+        self.assertEqual({{ case["property"] }}(input), expected)
```

Comment:
I'd recommend putting everything inside the `assertEqual` call instead of using the `input` and `expected` variables here. You can spread things out like so for template readability, and the generator will put everything on a single line in generation:
```
self.assertEqual(
    {{ case["property"] }}({{ case["input"]["sides"] }}),
    {{ case["expected"] }}
)
```